packages/expo-cli/src/appleApi/authenticate.ts

```diff
@@ -7,6 +11,99 @@ import prompt from '../prompt';
 import { nonEmptyInput } from '../validators';
 import { runAction, travelingFastlane } from './fastlane';
 
+const NO_STORE_PASSWORD = getenv.boolish('EXPO_NO_STORE_PASSWORD', false);
+
+function getKeychainServiceName(appleId: string): string {
+  return `deliver.${appleId}`;
```

Comment:
is `deliver` a convention or a shared namespace? Else we should prefix it with `expo` so we know its an entry from our services

Reply:
as mentioned in the PR description, this follows the Fastlane convention. I've also added a doc block to the method.